### PR TITLE
Remove FarmBeats

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md [hunk 1]
```diff
@@ -3,23 +3,6 @@
...
-  - AgriFoodFarmingFarmOperationDataIngestionJobStatusChangedEventData 
-  - AgriFoodFarmingFieldChangedEventData
-  - AgriFoodFarmingHarvestDataChangedEventData
-  - AgriFoodFarmingPlantingDataChangedEventData
-  - AgriFoodFarmingSatelliteDataIngestionJobStatusChangedEventData
-  - AgriFoodFarmingSeasonalFieldChangedEventData
-  - AgriFoodFarmingSeasonChangedEventData
-  - AgriFoodFarmingTillageDataChangedEventData
-  - AgriFoodFarmingWeatherDataIngestionJobStatusChangedEventData
-  - StorageBlobInventoryPolicyCompletedEventData
```
Comment (ellismg) [inline sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md:22]: StorageBlobInventoryPolicyCompletedEventData is still present.

**Review by ellismg — CHANGES_REQUESTED** [review]
`StorageBlobInventoryPolicyCompletedEventData` is still new in this release, right?

**Comment by JoshLove-msft** [comment]
> `StorageBlobInventoryPolicyCompletedEventData` is still new in this release, right?

Yep just caught this after looking at your PR.

**Review by ellismg — APPROVED** [review]
LGTM

**Pull request merged**